Users/models.py

```diff
@@ -26,5 +34,23 @@ def __str__(self):
 		return self.owner.username
 
 class Friend(models.Model):
-	FriendsList = models.ForeignKey(FriendsList, on_delete=models.CASCADE)
-	friend = models.OneToOneField(User, on_delete=models.CASCADE)
+
+	users = models.ManyToManyField(User, default=defaultUser)
+
+	#NEEDS REVIEW, not sure how to handle on_delete
+	current_user = models.ForeignKey(User, related_name='owner', null=True, on_delete=models.CASCADE)
+
+
+	@classmethod
+	def make_friend(cls, current_user, new_friend):
```

Comment:
Should the make_friend function also add the current_user to the new_friend's friend list?
Also, don't forget to write unit tests for these functions - let me know if you need help with this

Reply:
The Friend class is owned by the current user, and so when the Friend class is created it is linked to the respective person. When make_friend is called, the function grabs the friend model linked to the "current owner" of the user that called it. Therefore, all we need to do is add the "new_friend" to the list of users that are friends with the owner. I hope I explained that correctly, but basically, we could add the current_user as well, but it would just be duplicate work because the relationship is already linked to the current_user.